README.md

```diff
@@ -1,68 +1,154 @@
 
-- [File uploading solutions](/solutions/file-uploader/) — file uploader implementations for the most frequent use cases.
-- [Blocks to build custom solutions](/blocks/) — build your integrations with a simple HTML and CSS.
-- [Adaptive image](/solutions/adaptive-image/) — all you need to distribute your web images efficiently.
-- [Cloud image editor](/solutions/cloud-image-editor/) — tune your pictures with our CDN image processing right after they're uploaded.
-- Soon: web components for media content representation: audio, video, image galleries, etc.
+**Typescript support**
 
-## ⚙️ Installation and start
+We use [JSDoc type annotations](https://www.typescriptlang.org/docs/handbook/intro-to-js-ts.html) for TypeScript static analysis support during development.  Additionally, we provide type definitions ([\*.d.ts files](https://www.typescriptlang.org/docs/handbook/declaration-files/dts-from-js.html)) for the TypeScript projects in our packages.
 
-Installation process is easy and takes minutes. See the [installation guide](/get-started/installation/).
+## 🚀 Getting Started
 
-## Typescript support
+### HTML `<script>` Tag
 
-We use [JSDoc type annotations](https://www.typescriptlang.org/docs/handbook/intro-to-js-ts.html) for TypeScript static analysis support during development.
+* Connect `Blocks` directly from your document replacing `@{{PACKAGE_VERSION}}` with the [latest version](https://github.com/uploadcare/blocks/releases) of the package:
 
-Additionally, we provide type definitions ([\*.d.ts files](https://www.typescriptlang.org/docs/handbook/declaration-files/dts-from-js.html)) for the TypeScript projects in our packages.
+```html
+<script type="module">
+  import * as LR from 'https://cdn.jsdelivr.net/npm/@uploadcare/blocks@{{PACKAGE_VERSION}}/web/blocks.min.js';
+  
+  LR.registerBlocks(LR);
+</script>
+```
 
-Check the [JSDoc Reference](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html) page in TypeScript official documentation.
+* Start using `Blocks` in your application markup (don't forget to specify `@{{PACKAGE_VERSION}}` with the [latest one](https://github.com/uploadcare/blocks/releases)):
 
-## Supported browsers
+```html
+<lr-file-uploader-regular
+  css-src="https://cdn.jsdelivr.net/npm/@uploadcare/blocks@{{PACKAGE_VERSION}}/web/file-uploader-regular.min.css"
+  class="my-uploader"
+>
+</lr-file-uploader-regular>
+```
 
-Blocks are supported and tested in all major modern desktop and mobile browsers:
+* All configurations and customization in `Blocks` are managed from CSS code. Sign up to [Uploadcare](https://app.uploadcare.com/accounts/signup/) and get a Public API key in [Uploadcare project's dashboard](https://app.uploadcare.com/projects/-/api-keys/). Then add a style sheet to your project and replace `'YOUR_PUBLIC_KEY'` with your own public key:
 
-- Chrome
-- Firefox
-- Safari
-- Edge
-- Opera
+```css
+.my-uploader {
+  --ctx-name: 'my-uploader';
+  --cfg-pubkey: 'YOUR_PUBLIC_KEY';
+}
+```
```

Comment:
I see that this snippet is duplicated under "Using File Uploading Features" section. Maybe drop this one?

Reply:
I do wanna mention it here just in case the user focuses on this only section. However, I rewrote the text here and added the anchor link.